libs/auth-api-lib/migrations/20210519134445-api-scope-groups.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+/* eslint-env node */
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        CREATE TABLE api_scope_group (
+          id UUID NOT NULL,
+          name VARCHAR NOT NULL,
+          description VARCHAR NOT NULL,
+          created TIMESTAMP WITH TIME ZONE DEFAULT now(),
+          modified TIMESTAMP WITH TIME ZONE,
+          PRIMARY KEY (id)
+        );
+
+        ALTER TABLE api_scope 
+        ADD COLUMN api_scope_group UUID NULL;
```

Comment:
Maybe 'group' would be a more convenient name for the column in the 'api_scope' table?

Reply:
or group_id ?

Reply:
Sounds good